ui-lib/src/components/Divider/Divider.stories.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import Divider from "./index";
+import Text from "@ui/components/Text";
+
+export default {
+  title: "Divider",
```

Comment:
```suggestion
  title: "Interface/Divider",
```

To respect Figma structure as asked by Gregor.

_(we should find a new category name tbh)_